state.go

```diff
@@ -516,7 +605,16 @@ func NewRebootState(update UpdateResponse) State {
 	}
 }
 
-func (e *RebootState) Handle(c Controller) (State, bool) {
+func (e *RebootState) Handle(ctx *StateContext, c Controller) (State, bool) {
+	if err := StoreStateData(ctx.store, StateData{
+		Id:         e.Id(),
+		UpdateInfo: e.update,
+	}); err != nil {
+		// TODO: too late to do anything now, update is installed and enabled
```

Comment:
Hmm, maybe you're right, we've already reached reboot, so it's logical that install has completed with success and state data was written there.